### ISSUE TITLE
Localization for versafix template

### ISSUE THREAD

**Issue by dev-allinappli** [veda-consulting/uk.co.vedaconsulting.mosaico#392]
It took me some time to translate the versafix template into French for the 2.3 version and I'm wondering if I have to do this again for the new 2.4? Can I just copy paste or are there new options? Do you still integrate the version 1.1.19 from Voidlabs? Looking at the readme file seems to show that the template is not concerned but how can I be sure? Could you write this for the next version (something like 'based on the last versafix 1.1.XX version from voidlabs')?
thanks
Guillaume

**Comment by dev-allinappli** [comment]
not really an issue but what i the best way to interact about this point?

**Comment by mattwire** [comment]
Closing following 2.6 release. Please reopen if still an issue (we have pulled in the french translation for 2.6)

**Comment by dev-allinappli** [comment]
hi Matt, I think we're not talking about the same things. I've just updated Mosaico to version 2.6 and my issue remain the same. To use Mosaico in French I need two files:
- /mosaico/dist/lang/mosaico-fr.json
- /mosaico/templates/versafix-1/template-versafix-1.html

The .json file translates the page in order to create the campaign. I don't know where you got your file from because I can't find it in this repository and it is different from the "official" version at voidlabs ([https://github.com/voidlabs/mosaico/blob/master/res/lang/mosaico-fr.json](https://github.com/voidlabs/mosaico/blob/master/res/lang/mosaico-fr.json)). Your seems more recent. It is also different from the file I use and that it better translated into French, albeit it seems to lack one or two strings when comparing to the file you're providing. I would be happy to contribute if you send me the proper path.

The template-versafix-1.html is the source versafix template which I translated in order to have the template builder and options in French. I've uploaded this file and replace the standard one but I still find my template in English even though it's the only one in the folder. I've done this work following the recommendation of Voidlabs saying I have to do this translation inside the code of the master template. I also can't find this template in your repository although I want to translate it.

I think looking at my two files could help you to understand my issue. How could we solve this?

[Archive.zip](https://github.com/veda-consulting-company/uk.co.vedaconsulting.mosaico/files/5787616/Archive.zip)
